### PR TITLE
Replace the default port with 8080

### PR DESCRIPTION
**Pull request by sansyrox**
OSX has some service running on port 5000 by default. It makes it annoying to work on Robyn to keep changing the default port number all the time.

**Description**

This changes the default PORT of Robyn.

### CODE REVIEW

File: integration_tests/test_env_populator.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # create robyn.env before test and delete it after test
 @pytest.fixture
 def env_file():
-    CONTENT = """ROBYN_PORT=8080
+    CONTENT = """ROBYN_PORT=8081
```
Comment (AntoineRR) [inline integration_tests/test_env_populator.py:15]: Why using 8081 here?

**Review by AntoineRR — COMMENTED** [review]
Hi @sansyrox !

This looks fine to me and runs well on my linux machine. I just noticed there are a few places where port 5001 was used in the tests. Do you want to keep it like this? Maybe we should replace it with 8081 for consistency?

File: integration_tests/test_env_populator.py (hunk 1 above)
Comment (sansyrox) [reply integration_tests/test_env_populator.py:15]: @AntoineRR , as I wanted to test it on something other than the default port. Given that 8080 will be the default port from now on.

Comment (sansyrox) [reply integration_tests/test_env_populator.py:15]: >I just noticed there are a few places where port 5001 was used in the tests. Do you want to keep it like this?

Sure, this sounds like a good idea. 😄

**Comment by sansyrox** [comment]
@AntoineRR , all done. 😄

File: robyn/robyn.pyi [hunk 2]
```diff
@@ -21,6 +21,10 @@ class Response:
     headers: dict[str, str]
     body: str
 
+    def set_file_path(file_path):
```
Comment (AntoineRR) [inline robyn/robyn.pyi:24]: Can you add `str` typing to the parameter here?

**Review by AntoineRR — APPROVED** [review]
Sorry I forgot to add one comment about the typing. Otherwise this LGTM, approved :wink:

File: robyn/robyn.pyi (hunk 2 above)
Comment (sansyrox) [reply robyn/robyn.pyi:24]: @AntoineRR , yep sure 👍

Comment (sansyrox) [reply robyn/robyn.pyi]: done!

**Pull request merged**